openssl/src/ssl/mod.rs

```diff
@@ -282,6 +321,12 @@ impl SslContext {
         }
     }
 
+    pub fn set_read_ahead(&self) {
+        unsafe {
+            ffi::SSL_CTX_ctrl(*self.ctx, ffi::SSL_CTRL_SET_READ_AHEAD, 1, ptr::null_mut());
```

Comment:
This should apparently never be called directly: http://unixhelp.ed.ac.uk/CGI/man-cgi?SSL_CTX_ctrl+3